isovar/locus_reads.py

```diff
@@ -29,17 +28,34 @@
 )
 from .common import list_to_string
 from .dataframe_builder import DataFrameBuilder
-
+from .value_object import ValueObject
 
 logger = logging.getLogger(__name__)
 
-class LocusRead(namedtuple("LocusRead", [
+class LocusRead(ValueObject):
+    __slots__ = [
         "name",
         "sequence",
         "reference_positions",
         "quality_scores",
         "base0_read_position_before_variant",
-        "base0_read_position_after_variant"])):
+        "base0_read_position_after_variant"
+    ]
+
+    def __init__(
```

Comment:
@ihodes @julia326 

What I like about base classes like `ValueObject` is that I can treat everything in isovar like a functional record without having to write extra code. This might be a case of [DRY making things more confusing](http://thereignn.ghost.io/on-dry-and-the-cost-of-wrongful-abstractions/) (or, at least more confusing for other maintainers). So, I'm fine with `ValueObject` being a transitional state until a more verbose version of isovar comes along. Since we're aiming for eventual verbosity: maybe leave the `__init__` functions as is?